include/toml++/impl/node_view.h

```diff
@@ -719,6 +719,15 @@ TOML_NAMESPACE_START
 			return node_ ? node_->at_path(path) : node_view{};
 		}
 
+		/// \brief Returns a view of the subnode matching a fully-qualified "TOML path".
+		///
+		/// \see #toml::node::at_path(const toml::path&)
+		TOML_NODISCARD
+		node_view at_path(const toml::path& path) const noexcept
+		{
+			return node_ ? node_->at_path(path) : node_view{};
+		}
```

Comment:
Node views probably want an `operator[]` overload for paths, too. Being able to mix regular strings and toml::paths when chaining `[]` on node views seems like a pretty potent mix!

Reply:
This is a very cool idea, I like it!